start.js

```diff
@@ -17,6 +17,21 @@ if (!envExists) {
   .pipe(fs.createWriteStream(path.join(__dirname, '/.env')))
 }
 
+// Create template session data defaults file if it doesn't exist
+const sessionDefaultsDirectory = path.join(__dirname, '/app/data')
```

Comment:
I wonder if this should be named `defaultDataDirectory` - so that in the future if we reference files in this directory we can reuse it.

Reply:
+1 for this idea, I think that would be more suitable